tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_solve.md

```diff
@@ -193,6 +193,27 @@ during linear iterations.
 
 !---
 
+## Preconditioning
+
+Select a preconditioner using PETSC options, either in the executioner or in the `[Preconditioning]` block:
+
+```language=cpp
+[Executioner]
+  type = Steady
+  petsc_options_iname = '-pc_type -pc_hypre_type'
+  petsc_options_value = 'hypre boomeramg'
+```
+
+Some examples:
+
+- LU : form the actual Jacobian inverse, useful for small to medium problems but does not scale well
+- Hypre BoomerAMG : algebraic multi-grid, works well for diffusive problems
+- jacobi : preconditions with the diagonal of Jacobian
+
+Can also combine pre-conditioners using sub-preconditioning with option `-sub_pc_type`.
```

Comment:
```suggestion
For parallel solves, the sub-block (on-process) solvers can be controlled with the PETSc option `-sub_pc_type`. E.g. for a parallel block Jacobi preconditioner (-pc_type jacobi) the sub-block preconditioner could be set to ILU or LU etc. with `-sub_pc_type ilu`, `-sub_pc_type lu`, etc.
```
You can truly combine preconditioners with PETSc using `-pc_type composite`, but we don't really have first class support for this in MOOSE

Reply:
:-/